src/modules/Dropdown/Dropdown.js

```diff
@@ -747,7 +802,13 @@ export default class Dropdown extends Component {
         active={isActive(opt.value)}
         onClick={this.handleItemClick}
         selected={selectedIndex === i}
-        text={opt.text}
+        text={
+          allowAdditions
+          && (additionPosition === 'top' ? i === 0 : i === options.length - 1)
+          && opt.text.startsWith('Add: ') ?
+          <span><strong><em>Add:</em></strong> {opt.text.substring(5)}</span> :
+          opt.text
+        }
```

Comment:
With the update to `getMenuOptions`, this prop would also be reverted back to `text={text}`.  That is because the `getMenuOptions` method will be inserting an item with the text already set.

`renderOptions` gets its list of items from the `getMenuOptions`.